### PR TITLE
adds support for version 2015-04-11

### PR DESCRIPTION
**Pull request by mgartner**
Should pass once the new version is ready.

@pon @elnaz @robinjoseph08

### CODE REVIEW

File: lib/lob.rb [hunk 1]
```diff
@@ -27,6 +27,9 @@ def self.require_options(options, *keys)
   def self.submit(method, url, parameters={})
     clientVersion = Lob::VERSION
 
+    puts "submitting with version:#{self.api_version}"
+    puts "submitting with key:#{self.api_key}"
+
```
Comment (robinjoseph08) [inline lib/lob.rb]: should these `puts` still be here? looks like something left over from testing

File: lib/lob.rb [hunk 2]
```diff
@@ -67,6 +69,7 @@ def Lob(options={})
   options[:protocol]     ||= Lob.protocol    || "https"
   options[:api_version]  ||= Lob.api_version
   options[:api_key]      ||= Lob.api_key
+  puts "options: #{options[:api_version]}"
```
Comment (robinjoseph08) [inline lib/lob.rb]: should this be here

File: spec/lob/v1/job_spec.rb [hunk 3]
```diff
@@ -77,25 +76,25 @@
         settings_list = subject.settings.list
 
         new_object_params = {
-          name:    @sample_object_params[:name],
-          file:    "https://s3-us-west-2.amazonaws.com/lob-assets/test.pdf",
+          description:    @sample_object_params[:description],
+          file:    "https://lob.com/postcardfront.pdf",
           setting: 500
```
Comment (robinjoseph08) [inline spec/lob/v1/job_spec.rb]: fix alignment

File: spec/lob/v1/job_spec.rb [hunk 4]
```diff
@@ -77,25 +76,25 @@
...
         }
 
         another_object_params = {
-          name:    "Another #{@sample_object_params[:name]}",
-          file:    "https://s3-us-west-2.amazonaws.com/lob-assets/test.pdf",
+          description:    "Another #{@sample_object_params[:description]}",
+          file:    "https://lob.com/postcardfront.pdf",
           setting: 500
```
Comment (robinjoseph08) [inline spec/lob/v1/job_spec.rb]: alignment

File: spec/lob/v1/job_spec.rb [hunk 5]
```diff
@@ -106,32 +105,25 @@
         new_address = subject.addresses.create @sample_address_params
 
         new_object_params = {
-          name:    @sample_object_params[:name],
-          file:    File.new(File.expand_path("../../../samples/test.pdf", __FILE__)),
+          description:    @sample_object_params[:description],
+          file:    File.new(File.expand_path("../../../samples/postcardfront.pdf", __FILE__)),
           setting: 500
```
Comment (robinjoseph08) [inline spec/lob/v1/job_spec.rb]: alignment

File: spec/lob/v1/job_spec.rb [hunk 6]
```diff
@@ -106,32 +105,25 @@
...
         }
 
         another_object_params = {
-          name:    "Another #{@sample_object_params[:name]}",
-          file:    "https://s3-us-west-2.amazonaws.com/lob-assets/test.pdf",
+          description:    "Another #{@sample_object_params[:description]}",
+          file:    "https://lob.com/postcardfront.pdf",
           setting: 500
```
Comment (robinjoseph08) [inline spec/lob/v1/job_spec.rb]: alignment

File: spec/lob/v1/job_spec.rb [hunk 7]
```diff
@@ -162,18 +154,18 @@
         new_address = subject.addresses.create @sample_address_params
         settings_list = subject.settings.list
         new_object_params = {
-          name:      @sample_object_params[:name],
-          file:      "https://s3-us-west-2.amazonaws.com/lob-assets/test.pdf",
+          description:      @sample_object_params[:description],
+          file:      "https://lob.com/postcardfront.pdf",
           setting: @test_setting_id
```
Comment (robinjoseph08) [inline spec/lob/v1/job_spec.rb]: alignment

**Comment by mgartner** [comment]
@lob/api-engineering-team my tests are passing locally but not in travis. Something to do with full_bleed files. I'm at a loss. Any ideas?

**Comment by robinjoseph08** [comment]
The issue was with the sample files that were downloaded with `bundle exec rake dev:setup`. https://s3-us-west-2.amazonaws.com/lob-assets/postcardfront.pdf is 4x6.

**Comment by robinjoseph08** [comment]
Looks like the build is passing, but the coverage is still shy

**Comment by mgartner** [comment]
@robinjoseph08 you da man! :tada: 

I'll fix the coverage if need be. Thanks for finding that. That step should probably be removed anyway that seems unnecessary to me.

**Comment by robinjoseph08** [comment]
Yeah it looks like a lot of stuff in this gem is unnecessary. I just looked up what VCR does and that kinda seems like recording HTTP requests defeats the purpose of testing the gem since its main purpose is just communicating with the API.

**Comment by elnaz** [comment]
LGTM

**Comment by pon** [comment]
LGTM - there's a task for tomorrow to update the tests to all use HTML and data but this is good for now

**Pull request merged**